src/operation-node/insert-query-node.ts

```diff
@@ -23,6 +24,8 @@ export interface InsertQueryNode extends OperationNode {
   readonly replace?: boolean
   readonly explain?: ExplainNode
   readonly defaultValues?: boolean
+  readonly frontModifiers?: ReadonlyArray<InsertModifierNode>
+  readonly endModifiers?: ReadonlyArray<InsertModifierNode>
```

Comment:
I don't see why these should have their own nodes.
Should just be ReadonlyArray<OperationNode> for now.